src/app/(proper_react)/(redesign)/(authenticated)/admin/dev/page.tsx

```diff
@@ -10,11 +10,7 @@ import { UserAdmin } from "./UserAdmin";
 
 export default async function DevPage() {
   const session = await getServerSession();
-  if (
-    !session?.user?.email ||
-    !isAdmin(session.user.email) ||
-    process.env.APP_ENV === "production"
```

Comment:
This also doesn't seem limited to stage, and would expose this page in prod?

Reply:
All the actions are guarded and return a `404` when `process.env.APP_ENV === "production"`, except `triggerManualProfileScan` (callable for admin accounts only on production). We’ll need a way to trigger manual scans. I can keep the condition here for now (readded in [939fd31](https://github.com/mozilla/blurts-server/pull/5809/commits/939fd31f8d138bf0976a7c718a273fbdc5cd4fe9)) as a redundant safe-guard and add a new view with just a button for that (will do that in a follow-up after this has been tested on `stage`).